data-raw/typhoid_results.qmd

```diff
@@ -92,7 +73,10 @@ typhoid_results <- fit %>%
     ageCat = NULL,
     antigen.iso = paste(collapse = "+", "HlyE_IgG")
   ) %>%
-  structure(noise.parameters = cond.hlye.IgG)
+  structure(noise.parameters = noise)
 
 usethis::use_data(typhoid_results, internal = TRUE, overwrite = TRUE)
```

Comment:
see above; we now want to store this data in the testing subdirectories as described in <https://r-pkgs.org/testing-design.html#storing-test-data>

Reply:
It seems this link is not working.